packages/protocol/contracts/L1/tiers/TaikoA6TierProvider.sol

```diff
@@ -82,8 +80,14 @@ contract TaikoA6TierProvider is ITierProvider {
     }
 
     function getMinTier(uint256 rand) public pure override returns (uint16) {
-        if (rand % 1000 == 0) return LibTiers.TIER_SGX_AND_PSE_ZKEVM;
-        else if (rand % 100 == 0) return LibTiers.TIER_SGX;
+        // If we assume the block time is 3 seconds, and the proof generation
+        // time is ~90 mins and half of the blocks are unprovable:
+        // 90 * 60 / 3 / 2 = 900.
+        // TODO(david): tune this value based on the A6 circuits benchmark.
+        if (rand % 900 == 0) return LibTiers.TIER_SGX_AND_PSE_ZKEVM;
```

Comment:
This doesn't have to be really accurate.I prefer 1000 or 500, easier for manual calculation.

Reply:
I think @Brechtpd would prefer a higher number. so lets settle for 500 first then increate it if our SGX servers are not busy?

Reply:
Changed to `500`